settings.gradle

```diff
@@ -0,0 +1 @@
+if (JavaVersion.current() >= JavaVersion.VERSION_1_8) include ':java8'
```

Comment:
It's currently inferred from the name of the directory, and reinforced in settings.gradle. We can change those, and/or add the project.name property in the subproject's build.Gradle.